### PR TITLE
[Docs] PR for Document how to publish Node-IP as the broadcast address (Doc-PLAT-2639)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2639

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 1]
```diff
@@ -303,9 +303,68 @@ Continue configuring your Kubernetes provider by clicking **Add Region** and com
...
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - "yb-tserver"
+            topologyKey: kubernetes.io/hostname
+      podAnnotations:
+        prometheus.io/path: "/prometheus-metrics"
```
Comment (jvigil-yugabyte) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:335]: This `prometheus.io/path: "/prometheus-metrics"` annotation seems unrelated. Why is it included in the example? Otherwise, LGTM

**Review by jvigil-yugabyte — APPROVED** [review]

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:335]: @bhavin192 Could you please, let me know if I can delete
    ```
  podAnnotations:
        prometheus.io/path: "/prometheus-metrics"
```

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 2]
```diff
@@ -303,9 +303,68 @@ Continue configuring your Kubernetes provider by clicking **Add Region** and com
     ```yml
     istioCompatibility: enabled: true
     ```
+    
+  - Overrides to publish Node-IP as the server broadcast address.
+  
+    By default, master and T-Server pod IPs are published as the server broadcast address. To publish the IPs of the nodes on which database pods are deployed, add the following YAML to each zone override configuration:
```
Comment (bhavin192) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:309]:
```suggestion
    By default, Master and T-Server pod FQDNs are used within the cluster as the server broadcast address. To publish the IPs of the nodes on which database pods are deployed, add the following YAML to each zone override configuration:
```

Comment (bhavin192) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:309]: > To publish the IPs of the nodes on which database pods are deployed,

The override we have here is actually exposing only YB-TServer pods and not master.

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 1 above)
Comment (bhavin192) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:335]: Yes, we can remove it Jason is correct.

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md [hunk 3]
```diff
@@ -303,9 +303,68 @@ Continue configuring your Kubernetes provider by clicking **Add Region** and com
...
+  
+    ```yml
+    master:
+      podAnnotations:
+        prometheus.io/path: "/prometheus-metrics"
+        # Any other required annotations
+
```
Comment (bhavin192) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:316]: This whole section can be removed as well. Same as the suggestion given by Jason for tserver section.

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 1 above)
Comment (bhavin192) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:335]: Yes, we can remove it Jason is correct.

**Review by bhavin192 — APPROVED** [review]
I have added few suggestions, LGTM otherwise.

**Comment by lizayugabyte** [comment]
@bhavin192 could you, please, get back to me on Jason's comment

**Comment by bhavin192** [comment]
@lizayugabyte I have replied there, not sure if it is visible or not (the UI is a bit confusing I think).
- https://github.com/yugabyte/yugabyte-db/pull/11056#discussion_r783672927
- https://github.com/yugabyte/yugabyte-db/pull/11056#discussion_r783673636

We can remove the blocks as Jason said, let me add suggestions, so that it is easier for you.

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:309]: Implemented the suggestion

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md:316]: Removed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md (hunk 2 above)
Comment (bhavin192) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md]: @lizayugabyte that's correct.

**Review by bhavin192 — APPROVED** [review]
